e2e/bzlmod/BUILD.bazel

```diff
@@ -57,3 +57,8 @@ build_test(
     name = "node_modules_test",
     targets = ["//:node_modules"],
 )
+
+build_test(
+    name = "other_module_binary_test",
+    targets = ["@other_module//:pyright"],
```

Comment:
This failed here before your fix? The scenario is a `js_binary` generated from `npm_translate_lock` which is referenced from a different module/workspace?

Can we add a minor comment saying that here? (otherwise I'll look at this in 1y and completely forget what it is testing :|).

Reply:
Done!